docs/source/tutorials/building_blocks.rst

```diff
@@ -291,7 +290,7 @@ Use the :code:`run_time` argument to control the duration.
        def construct(self):
            square = Square()
            self.add(square)
-	   self.play(ApplyMethod(square.shift, UP), run_time=3)
+	   self.play(square.animate().shift(UP), run_time=3)
```

Comment:
For some reason this also works, but I think we should at least have consistent indentation within an example.

Reply:
I accidentally made a mixed use of tab and spaces.  I have fixed it.  Sorry about that.